packages/types/src/codec/createType.ts

```diff
@@ -10,6 +10,7 @@ import Compact from './Compact';
 import Tuple from './Tuple';
 import UInt from './UInt';
 import Vector from './Vector';
+import TypeRegistry from './TypeRegistry';
```

Comment:
There is actually - we have no enabled it since we have a funny import structure, i.e.

1. Any externals
2. Our packages (Anything in the `@polkadot` space)
3. Then locals

So the order of 1 & 2 above is where it gets messy, it will always put the `@polkadot` stuff first, because of the namespace. I'm leaning towards enabling that one and then just go and fix all the projects (each and every file will break atm.)

So yes, there are tools, but we got it messy since it wasn't enabled.